cloud/start_batch_import.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# This script packages up the importer as it exists in the current git repo,
+# deploys it to AWS Batch, and regenerates maps and scenarios for all cities.
+#
+# This process is only runnable by Dustin, due to current S3/EC2 permissions.
+#
+# Run from the repo's root dir: cloud/workflow.sh
+
+set -e
+set -x
+
+EXPERIMENT_TAG=$1
+if [ "$EXPERIMENT_TAG" == "" ]; then
+	echo Missing args;
+	exit 1;
+fi
+
+# It's a faster workflow to copy the local binaries into Docker, rather than
+# build them inside the container. But it does require us to build the importer
+# without the GDAL bindings, since the dynamic linking won't transfer over to
+# the Docker image.
```

Comment:
alternatively could you install gdal in the docker container?

Reply:
Yes, there's even an `osgeo/gdal` image with it already set up. The problem is that the importer binary built on my local machine is dynamically linked to some version of libgdal.so, and copying the binary into Docker, the linking doesn't work. Maybe if the version of libgdal was the same, it might work? The "proper" alternative is building the importer inside of Docker, but I don't feel like it's worth the build time. We only need the GDAL dep to convert coordinates for PSRC scenario data, and I've only ever done that process a few times. Most import runs just use the cached file.